src/teaching-element/Assessment/MatchingQuestion.vue

```diff
@@ -117,6 +118,17 @@ export default {
     draggable(item) {
       return get(item.answers, '0.value', '');
     },
+    retakeIncorrect() {
+      this.target.forEach(response => {
+        const key = response.answers[0].key;
+        if (this.correct[key] !== response.key) {
+          this.source.map(it => {
```

Comment:
`map` returns a new array which is not used here so there is no need to use it here. `forEach` should be used instead.